codecept.conf.js

```diff
@@ -6,6 +6,11 @@ exports.config = {
   multiple: {
     parallel: {
       chunks: (files) => {
+        // const skipFlakyTests = parseInt(process.env.SKIP_FLAKY_TESTS || 0);
+        //
+        // if (skipFlakyTests === 1) {
+        //   files = lodash.filter(files, file => !file.includes('@flaky'))
+        // }
```

Comment:
do we need to keep this bit? 

otherwise though LGTM

Reply:
ah yes sorry, forgot about this. there's another ref i need to remove too :+1: